lms/djangoapps/courseware/views.py

```diff
@@ -739,6 +739,24 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        #Translators: This may look like $50 or ¥900, while in some languages it may look like 50$ or 900¥.
```

Comment:
Please put a space between "#" and the start of a comment line. Also this line has a Pylint error. Run `paver run_quality` locally and make sure it's 100% or else the quality shard fails.